lib/App/Asciio/Actions.pm

```diff
@@ -27,22 +28,33 @@ my ($self, $keep_visible) = @_ ;
 if($self->{USE_BINDINGS_COMPLETION})
 	{
 	my %reserved = map { $_ => 1 } qw(IS_GROUP ENTER_GROUP ESCAPE_KEYS NAME SHORTCUTS ORIGIN CODE) ;
+	my $hide_bindings_name = $self->{HIDE_GROUP_BINDING_HELP} // {} ;
+	my $current_actions = Clone::clone($self->{CURRENT_ACTIONS}) ;
 
```

Comment:
HIDE_GROUP_BINDING_HELP is at the wrong place! it should not be part of a config but in the group itself,

open a ticket for me to implement it